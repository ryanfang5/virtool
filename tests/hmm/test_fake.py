async def test_create_fake_hmm(fake2, snapshot):
    hmm = await fake2.hmm.create(fake2.mongo)

    assert hmm == snapshot
