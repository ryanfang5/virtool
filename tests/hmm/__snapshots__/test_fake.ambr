# name: test_create_fake_hmm
  <class 'dict'> {
    'cluster': 3088,
    'count': 9934,
    'entries': <class 'list'> [
      <class 'dict'> {
        'accession': 'yEVAoNGnVZQUqLUJyfwF',
        'gi': 'VYySnPCaLuQIazTmqTjD',
        'name': 'mYPxeqAWfCKCQCYFExFu',
        'organism': 'DpjjFIyeNTWRUWCuKoQS',
      },
    ],
    'families': <class 'dict'> {
      'RgeRgFkYLyLdJCGRddpw': 1000,
    },
    'genera': <class 'dict'> {
      'UEXExIZVPeFzGRmMglPr': 1018,
    },
    'id': 'RNvnAvOp',
    'length': 1648,
    'mean_entropy': 89.795508575047,
    'names': <class 'list'> [
      'AcfZksMRViIYdJRrnFxj',
    ],
    'total_entropy': 0.89365919009586,
  }
---
